book/src/install.md

```diff
@@ -61,6 +61,11 @@ Helix also needs it's runtime files so make sure to copy/symlink the `runtime/`
 config directory (for example `~/.config/helix/runtime` on Linux/macOS). This location can be overriden
 via the `HELIX_RUNTIME` environment variable.
 
+| OS        | command   |
+|-----------|-----------|
+|windows    |`xcopy runtime %AppData%/helix/runtime`|
+|linux/macos|`ln -s $PWD/runtime ~/.config/helix/runtime`
```

Comment:
Could you copy over this table to the readme on line 51? I don't mind it being duplicated between the two